nfc/doc/integration_notes.rst

```diff
@@ -23,15 +25,13 @@ Integration notes
   * HFCLK is running.
 
 * Exemplary implementation of the NFC Platform module for the Zephyr environment
-  is provided in the nfc_platform_zephyr.c file, which is located in the nfc/src
-  folder.
+  is provided in the nRF Connect SDK :file:`subsys/nfc/lib/platform.c` file.
```

Comment:
@greg-fer Can we do a link here to NCS file ?

Reply:
We probably do not have a rst file for it in the NCS.

Reply:
Then maybe we should add one - this would help keep the documentation up to date. Currently when the port changes in _fw-nrfconnect-nrf_, documentation needs to be updated in _nrfxlib_, but let's leave this change for future, it's not in the scope of this PR.